a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -1057,6 +1052,8 @@ def execute(self, vthunder, use_device_flavor):
             try:
                 member_list = []
                 members = []
+                if vthunder.partition_name != 'shared':
```

Comment:
I thought this will be done in axapi_client_decorator(), is there a reason we need to active the partition again here?

Reply:
This is for activating the l3v partition and getting member count specific to a subnet on that **particular partition** if user is using l3v partition instead of shared.
This will handle the case of VRID FIP deletion for a member on l3v partition.

Reply:
Removed this axapi call